src/Microsoft.Framework.OptionsModel/project.json

```diff
@@ -6,6 +6,10 @@
     "Microsoft.Framework.DependencyInjection.Abstractions": "1.0.0-*",
     "Microsoft.Framework.NotNullAttribute.Sources": { "type": "build", "version": "1.0.0-*" }
   },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/aspnet/options"
```

Comment:
Nope. It picks everything else when it builds. But you must build from a folder that's a git repo

Sent from my phone
________________________________
From: Eilon Lipton<mailto:notifications@github.com>
Sent: ‎7/‎3/‎2015 11:10 AM
To: aspnet/Options<mailto:Options@noreply.github.com>
Cc: Victor Hurdugaci<mailto:victor.hurdugaci@microsoft.com>
Subject: Re: [Options] Add repository information to packages (#71)


In src/Microsoft.Framework.OptionsModel/project.json<https://github.com/aspnet/Options/pull/71#discussion_r33876560>:

> @@ -6,6 +6,10 @@
>      "Microsoft.Framework.DependencyInjection.Abstractions": "1.0.0-*",
>      "Microsoft.Framework.NotNullAttribute.Sources": { "type": "build", "version": "1.0.0-*" }
>    },
> +  "repository": {
> +    "type": "git",
> +    "url": "https://github.com/aspnet/options"


So how does this really work? It doesn't need a branch or tag or something?

—
Reply to this email directly or view it on GitHub<https://github.com/aspnet/Options/pull/71/files#r33876560>.